pkg/pool-weighted/test/foundry/CircuitBreakerLib.t.sol

```diff
@@ -0,0 +1,144 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+pragma solidity ^0.7.0;
+pragma experimental ABIEncoderV2;
+
+import { Test } from "forge-std/Test.sol";
+
+import "../../contracts/lib/CircuitBreakerLib.sol";
+
+contract CircuitBreakerLibTest is Test {
+    using FixedPoint for uint256;
+
+    uint256 private constant _MINIMUM_BOUND_PERCENTAGE = 1e17;  // 0.1
+    // The weight complement (1 - w) is bounded by the min/max token weights
+    uint256 private constant _MINIMUM_TOKEN_WEIGHT = 1e16; // 0.01 (1%)
+    uint256 private constant _MAXIMUM_TOKEN_WEIGHT = 99e16; // 0.99 (99%)
+    uint256 private constant _MAX_BOUND_PERCENTAGE = 2e18; // 2.0
+    uint256 private constant _MIN_BPT_PRICE = 1e6;
+
+    uint256 private constant _MAX_RELATIVE_ERROR = 1e16;
+    uint256 private constant _MAX_BPT_PRICE = type(uint112).max;
```

Comment:
This is a bit extreme - it corresponds to a price of 10^15

Reply:
What do you think it should be? 96 bits? That's about 10^10